src/components/PurchaseOrder/PO.js

```diff
@@ -137,8 +153,16 @@ class PO extends Component {
   }
 
   onAddPOLine = (e) => {
+    const { resources } = this.props;
+    const linesLimit = Number(get(resources, ['linesLimit', 'records', '0', 'value'], '1'));
```

Comment:
Firstly, `'1'` makes not much sense to me.
Secondly - why back-end doesn't return a number, since it's a number-typed config?

Reply:
in mod-config schema value is defined as a string
https://s3.amazonaws.com/foliodocs/api/mod-configuration/config.html#configurations_entries_post